gst/nnstreamer/tensor_demux/gsttensordemux.c

```diff
@@ -375,10 +394,40 @@ gst_tensor_demux_get_tensor_pad (GstTensorDemux * tensor_demux,
   g_free (stream_id);
   gst_event_unref (event);
 
-  caps = gst_tensor_caps_from_config (&config);
-  gst_pad_set_caps (pad, caps);
-  gst_element_add_pad (GST_ELEMENT_CAST (tensor_demux), pad);
+  if (tensor_demux->tensorpick != NULL) {
+    gchar *seleted_tensor =
+        (gchar *) g_list_nth_data (tensor_demux->tensorpick, nth);
```

Comment:
Is it ensured that `tensorpick` always has `nth` data?

Reply:
Right. I added error handling code (Line 403).